swift-ios/Task Tracker/AppDelegate.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AppDelegate.swift
+//  Task Tracker
+//
+//  Created by MongoDB on 2020-04-30.
+//  Copyright © 2020 MongoDB, Inc. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+
+let app = RealmApp("myrealmapp-vjmee", configuration: AppConfiguration(baseURL: "https://realm-dev.mongodb.com",
```

Comment:
can we extract these strings and others throughout the app into a constants file? It will be easier to have one place to update them all and should be easier for users to modify for their own stuff.

Reply:
Ah yes, should have done that. Thanks!